docs/demo/assets/styles/partials/_sg-breakpoint.scss

```diff
@@ -1,42 +1,31 @@
 //  ---------------------------------------------------------------------------
 //
-//  Demoes the different breakpoints.
+//  Demos the different breakpoints.
 //
-//	Breakpoints:   0px -> Tiny
-//				 320px -> Small
-//				 480px -> Medium
-//				 600px -> Large
-//				1024px -> Huge
-//				1280px -> Giant
-//              1600px -> Monstrous
+//  small : 320px {Smartphones}
+/// medium : 600px {Tablets}
+/// large : 1024px {Desktops}
+/// huge : 1280px {Full HD}
 //
 //  ---------------------------------------------------------------------------
 
 .dc-body--demo-breakpoints {
     @include dc-debugmessage("Tiny \2014 0-320px", $dc-red60);
 
-    @include breakpoint($dc-small-width) {
-        @include dc-debugmessage("Small \2014 320-480px", $dc-orange60);
+    @media (min-width:$dc-small-width) {
+        @include dc-debugmessage("Small \2014 320-600px", $dc-orange60);
```

Comment:
Wouldn't this be `599px` instead of `600px` if your next breakpoint has min-width?